ote/src/cljs/ote/ui/main_header.cljs

```diff
@@ -264,8 +264,11 @@
          [:li
           [linkify (tr [:common-texts :navigation-feedback-link]) (tr [:common-texts :navigation-give-feedback])
            (merge (stylefy/use-style
-                    style-topnav/topnav-dropdown-link)
-                  {:target "_blank"})]]]]]]]))
+                    (merge style-topnav/topnav-dropdown-link
+                           {:padding "10px 0 0 0"}))
```

Comment:
padding `rem` instead of `px` ?

Reply:
It's the same exact spacing used in other top navigation elements, so i didn't want to change it everywhere.